apps/docs/src/.vitepress/env.ts

```diff
@@ -1,5 +1,3 @@
 export const vitepressEnv = {
-  storybookHost:
-    import.meta.env.VITE_STORYBOOK_HOST ||
-    "https://onyx-storybook-dev.apps.01.cf.eu01.stackit.cloud",
+  storybookHost: import.meta.env.VITE_STORYBOOK_HOST || "https://storybook.onyx.schwarz",
```

Comment:
As we discussed in #60 we will route the regular `storybook.onyx.schwarz` to the dev stage until we have a prod stage in the future so this is fine for now.

But when we have a second stage, it will be replaced in the pipeline yes